cpp/src/arrow/builder_benchmark.cc

```diff
@@ -458,6 +496,8 @@ BENCHMARK(BuildBinaryArray);
 BENCHMARK(BuildChunkedBinaryArray);
 BENCHMARK(BuildFixedSizeBinaryArray);
 BENCHMARK(BuildDecimalArray);
+BENCHMARK(BuildInlineBinaryViewArray);
+BENCHMARK(BuildNonInlineinaryViewArray);
```

Comment:
Builds are failing because of this typo of mine. Sorry.

```suggestion
BENCHMARK(BuildNonInlineBinaryViewArray);
```